solutions/instances/variables.tf

```diff
@@ -135,6 +147,16 @@ variable "enable_platform_metrics" {
   default     = true
 }
 
+########################################################################################################################
+# AT targets and routes
+########################################################################################################################
+
+variable "provision_atracker" {
+  description = "Set it to true if AT targets and routes needs to be configured."
+  type        = bool
+  default     = true
+}
```

Comment:
Variable name needs update, it shows as if DA is provisioning AT.
We can keep something like - configure_cos_target_and_route_for_atracker_events
Description - Set it to true if COS targets and routes needs to be configured for activity tracker event routing

Reply:
changed